lisa/sut_orchestrator/ready.py

```diff
@@ -68,6 +85,11 @@ def _delete_environment(self, environment: Environment, log: Logger) -> None:
         # ready platform doesn't support delete environment
         pass
 
+    def delete_environment(self, environment: Environment) -> None:
+        # ready platform doesn't support delete environment
```

Comment:
This comment is confusing. Please write the logic about why it needs to prevent deleting dirty env in some cases.